### ISSUE TITLE
"service: enable=yes/no" with spanish CentOS6 servers is broken

### ISSUE THREAD

**Issue by ansibot** [ansible/ansible#29818]
From @vicendominguez on 2016-04-14T16:46:43Z
##### ISSUE TYPE
- Documentation Report
##### COMPONENT NAME

service core module
##### ANSIBLE VERSION

```
ansible 2.0.1.0 
(but I wouldn't discard other versions)

```
##### OS / ENVIRONMENT

CentOS 6.7 
(at least)
##### SUMMARY

On CentOS6 at least, a spanish translator did a bad translating in some  error messages returned to the command line by the chkconfig tool which break the "service enable=yes / no" option in Ansible.
It 's not a direct Ansible issue but Ansible is affected.
##### STEPS TO REPRODUCE

```
- name: Start myservice and enable it
  service: name=myservice state=started enabled=yes
```
##### EXPECTED RESULTS

```
TASK [role : Start myservice and enable it ] ***********************************************
ok: [myservice]
```
##### ACTUAL RESULTS

```
TASK [role : Start myservice and enable it ] ***********************************************
fatal: [myservice]: FAILED! => {"changed": false, "failed": true, "msg": "service myservice does not support chkconfig"}
```
##### WHY

Line https://github.com/ansible/ansible-modules-core/blob/devel/system/service.py#L702 waits a concrete answer to do the regexp but, in Spanish, that line is bad translated changing the "--add" statement to "--añada" (doesn't make sense), so you see something like:

"El servicio myservice soporta chkconfig, pero no está registrado (ejecute \n'chkconfig -- añada myservice')\n"

the sentence "chkconfig --add" doesn't appear there and that breaks the "if". So the "enable" feature is broken.
##### WORKAROUND

`export LANG="en"` 

in the terminal, before starting to work with any sofware which uses Ansible inside.

Copied from original issue: ansible/ansible-modules-core#3438

**Comment by ansibot** [comment]
From @ansibot on 2016-04-14T16:46:43Z



@grossws ping, this issue is waiting for your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by biomassives** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  acmeideal@gmail.com
    Domain biomassiv.es has exceeded the max emails per hour (121/100 (121%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
------ The body of the message is 6645 characters long; only the first
------ 5000 or so are included here.
Received: from github-smtp2-ext2.iad.github.net ([192.30.252.193]:34478 helo=github-smtp2b-ext-cp1-prd.iad.github.net)
	by chi-server32.websitehostserver.net with esmtps (TLSv1.2:ECDHE-RSA-AES256-GCM-SHA384:256)
	(Exim 4.89)
	(envelope-from <noreply@github.com>)
	id 1drZKS-001rpj-Gl
	for greg@biomassiv.es; Mon, 11 Sep 2017 19:45:28 -0500
Date: Mon, 11 Sep 2017 17:44:48 -0700
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=github.com;
	s=pf2014; t=1505177088;
	bh=RCSJT2df6A+CFMfFW0ODz/ipbybGCx1Donpa79uost8=;
	h=From:Reply-To:To:Cc:Subject:List-ID:List-Archive:List-Post:
	 List-Unsubscribe:From;
	b=fo80yNKel9tMIzDZIxyJp76CfteMEwnqfCREucjhRKqz27c+rXl+emxGEgoDbv6Xn
	 kHY5ot+BoN1Wuno8b0KkfPTC1QpGwg9Xav+jOy2qpuMnmUoj85/QMXJ6ghtpXhLlQD
	 CCk4X8iEkwHmg6/Zrf8PQTOw4Gv8agq/M0gzh90g=

**Comment by biomassives** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  acmeideal@gmail.com
    Domain biomassiv.es has exceeded the max emails per hour (122/100 (122%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from github-smtp2-ext6.iad.github.net ([192.30.252.197]:34824 helo=github-smtp2b-ext-cp1-prd.iad.github.net)
	by chi-server32.websitehostserver.net with esmtps (TLSv1.2:ECDHE-RSA-AES256-GCM-SHA384:256)
	(Exim 4.89)
	(envelope-from <noreply@github.com>)
	id 1drZKT-001rqf-RI
	for greg@biomassiv.es; Mon, 11 Sep 2017 19:45:29 -0500
Date: Mon, 11 Sep 2017 17:44:49 -0700
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=github.com;
	s=pf2014; t=1505177089;
	bh=Cw9LnNPmthMKRGJ/fuElROaLssfpSodX40g8DUPL1s4=;
	h=From:Reply-To:To:Cc:In-Reply-To:References:Subject:List-ID:
	 List-Archive:List-Post:List-Unsubscribe:From;
	b=r7l5+I6ZuYkl/x1R6PSTMh1FTiVBTwa/YBl6DnciUd1CUFNO+EBjH7FyBAdXdO2u4
	 EDFgOtm03jrn4JkDnH6Jai1CBvmisvcBUJ8mEVYtJAw3OEx9w5IjzqNHi0dDqDRHtc
	 deHkSj73iNciQ4ZPVhwXUnR/wYWK/wC7DsyrQ+wA=

**Comment by adrianlzt** [comment]
[service.py](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/system/service.py#L734) should force the locale before checking the stdout/err of the chkconfig command as suggested by the deprecation message in DEFAULT_MODULE_LANG: ``Modules are coded to set their own locale if needed for screenscraping``
